src/fedora_easyfix/utilities/producer.py

```diff
@@ -46,7 +47,10 @@ def __init__(self):
         self.gitlab_api_key = self.envrvars["GITLAB_API_KEY"]
         self.rplist_url = self.envrvars["RPLIST_URL"]
         self.yamldict = load(httpobjc.request("GET", self.rplist_url).data.decode(), Loader=CLoader)
-        self.ticket_collection = {}
+        self.ticket_collection = {
+            "forges": {},
+            "collection_updated_at": 0.0
```

Comment:
You never seem to set `collection_updated_at` to a concrete value, remove it?

Reply:
Here, https://github.com/t0xic0der/fedora-easyfix/blob/9f3bff0add8a39a24deadffacc2f6433035c379f/src/fedora_easyfix/utilities/producer.py#L58

Reply:
Ahh!